app/services/benefit_check_service.rb

```diff
@@ -1,7 +1,15 @@
 class BenefitCheckService
+  class BenefitCheckError < StandardError; end
+
   BENEFIT_CHECKER_NAMESPACE = "https://lsc.gov.uk/benefitchecker/service/1.0/API_1.0_Check".freeze
```

Comment:
Nit: If var used only once, would it be ideal to add it to  the below `NAMESPACES`

Reply:
👍 have added to NAMESPACES